File: modul 7/soal 3/PRAK603-2210817220034-Wanda Karliyanti.c
#include<stdio.h>

int main(){
    int b1, b2, i, a, h;
    scanf("%d %d", &b1, &b2);
    int angka1[b1], angka2[b2];
    if(b1==b2){
    for(i=0;i<b1;i++){
        scanf("%d", &a);
        angka1[i]=a;
    }
    for(i=0;i<b2;i++){
        scanf("%d", &a);
        angka2[i]=a;
    }
        for(i=0;i<b1;i++){
        h=angka1[i]*angka2[i];
        printf("%d ", h);
    }
    }
    else{
        printf("Jumlah Tidak Sama");
        }
}

File: modul 7/soal 3/PRAK603-2210817220034-Wanda Karliyanti.py
b1, b2 = map(int, input().split())
if (b1!=b2):
    print("\nJumlah tidak sama")
else:
    angka1=list(map(int, input().split()))
    angka2=list(map(int, input().split()))
    print("")
    for i in range (b1):
        print(angka1[i]*angka2[i], end=' ')